sequence diagrams using ascii arts (#30)
peppelinux/federation-wallet: openid-federation-wallet-1_0.md

| Wallet Provider       | `federation_entity`, `openid_wallet_provider`              | this specification                                  |
| Authorization Server  | `federation_entity`, `oauth_authorization_server`          | [@!OpenID4VCI], [@!RFC8414]                    |
| Credential Issuer     | `federation_entity`, `openid_credential_issuer`, `oauth_authorization_server` | [@!OpenID4VCI], this specification |
| Credential Verifier   | `federation_entity`, `openid_credential_verifier`          | [@!OpenID.Federation], [@!OpenID4VP], this specification       |

> **Table 1**: Map of the Federation Entity Types and corresponding metadata types for the Wallet architectures.

Author: ```suggestion
**Table 1**: Map of the Federation Entity Types and corresponding metadata types for the Wallet architectures.

```

| Authorization Server  | `federation_entity`, `oauth_authorization_server`          | [@!OpenID4VCI], [@!RFC8414]                    |
| Credential Issuer     | `federation_entity`, `openid_credential_issuer`, `oauth_authorization_server` | [@!OpenID4VCI], this specification |
| Credential Verifier         | `federation_entity`, `openid_credential_verifier`         | [@!OpenID.Federation], [@!OpenID4VP], this specification       |

> **Table 1**: Federation Trust Chain and Wallet Attestation are separate things, where the Wallet Attestation is linked to its Issuer attested within the Trust Chain.

Author: ```suggestion
**Table 1**: Map of the Federation Entity Types and corresponding metadata types for the Wallet architectures.
```

|           |                                                     |              |                |        |
|           |----+                                                |              |                |        |
|           |    | Include CI in Discovery Page                   |              |                |        |
|           |    | with validated information and logo            |              |                |        |
|           |<---+                                                |              |                |        |
+-----------+-----------------------------------------------------+--------------+----------------+--------+
        +---+--+                                           +------+-----+ +------+-----+ +--------+--------+          
        |Wallet|                                           |Trust Anchor| |Intermediate| |Credential Issuer|          
        +------+                                           +------------+ +------------+ +-----------------+    
~~~
**Figure 4**: Federation Credential Issuer listing, the Wallet Instance browse the entire federation collecting all the Credential Issuers.  

> The diagram above shows how a Wallet navigates the federation, collecting and validating the Trust Chain for each Credential Issuer (CI), and creating a discovery page including each Credential Issuer using the information, such as the credential types and logo obtained through their Trust Chain.

Author: ```suggestion
The diagram above shows how a Wallet navigates the federation, collecting and validating the Trust Chain for each Credential Issuer (CI), and creating a discovery page including each Credential Issuer using the information, such as the Credential Types and logo obtained through their Trust Chain.
```
